Gemfile

```diff
@@ -5,6 +5,8 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in masamune.gemspec
 gemspec
 
+gem "yarp", "0.13.0"
```

Comment:
I think this should go in the `gemspec` otherwise gems using masamune will break.

```ruby
spec.add_dependency "prism", ">= 0.13.0"
```

Note: I recommend keeping the version open ended too so app's Gemfiles can control the version, rather than us needing to ship a lot of Masamune updates to get newer Prism versions supported.

Reply:
Updated in https://github.com/gazayas/masamune-ast/pull/60/commits/5a4a68e369c830a64666eb9e50c9ef9f7721a5b9